docs/changelog.md

```diff
@@ -6,16 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## Upcoming
 
-### Added
+## 5.0.0-rc.2 - 2025-08-28
 
 ### Fixed
```

Comment:
The rating component was added :) this should be mentioned as well

Reply:
Wasn't that added in RC.1? See https://six-group.github.io/six-webcomponents/v5/changelog.html